tests/scripts/all.sh

```diff
@@ -1290,6 +1290,42 @@ component_test_no_use_psa_crypto_full_cmake_asan() {
     if_build_succeeded env OPENSSL_CMD="$OPENSSL_NEXT" tests/compat.sh -e '^$' -f 'ARIA\|CHACHA'
 }
 
+component_test_psa_crypto_config_basic() {
+    # full plus MBEDTLS_PSA_CRYPTO_CONFIG, MBEDTLS_PSA_CRYPTO_DRIVERS,
+    # and PSA_CRYPTO_DRIVER_TEST minus MBEDTLS_USE_PSA_CRYPTO
+    msg "build: full config plus MBEDTLS_PSA_CRYPTO_CONFIG MBEDTLS_PSA_CRYPTO_DRIVERS"
+    msg "build: minus MBEDTLS_USE_PSA_CRYPTO"
```

Comment:
Please keep the message on a single line so that each `msg` call describes what happens between it and the next `msg` call. It's ok to go a little over 80 columns. The message doesn't have to be sufficiently detailed to re-create the configuration, it only needs to be precise enough to identify the (step of the) component. For example: “full + MBEDTLS_PSA_CRYPTO_CONFIG” (since the other two options are, at least for now, not tested differently).